src/internal/observable/empty.ts

```diff
@@ -4,6 +4,36 @@ import { SchedulerLike } from '../types';
 /**
  * The same Observable instance returned by any call to {@link empty} without a
  * `scheduler`. It is preferrable to use this over `empty()`.
+ *
+ * ## Examples
```

Comment:
I find both of these examples confusing... but I do want to thank you for pointing out that we need good examples of when to use `EMPTY`. You can remove these examples, and we'll get the fix to the link merged. (Unless @JWO719 can come up with a better fix for the links)

Reply:
Agreed but couldn’t think of something small and clear, at least this got your attention.

Tried to come up with clear real life examples but when I wrote it I realized I introduced lots of other operators which confuses people who actually need to read these